main/app/attentionRelated/ears/conf/client.ini

```diff
@@ -2,3 +2,4 @@ subsystems          (recog)
 reachingRangeMin    (-0.5 -0.3 -0.15)
 reachingRangeMax    (-0.1 0.3 0.5)
 robot               icub
+zOffset             0.0
```

Comment:
The connection to `ARE` should not be an issue. The subsystems work surprisingly well. And if the vision is not well calibrated, everything else won't work either. So I'm in favor of removing `zOff` / `zOffset`.